salesagility/SuiteCRM
# Scrm 585 flag btn

## Description



Adding: 
- mark as flag buttons to bulk action onto email list view
- tidy up indicators style
- fix uids at checkboxes
(ref: SCRM-585, #3610 )

## Motivation and Context

Mark as Flag/Unflagged/Read/Unread function are missing

## How To Test This

- rebuild style.css (sass)
- QRR
- check emails list view => bulk action => "marks as ..." menuitems

## Types of changes


### Final checklist
